apps/drec-ui/src/main/CustomErrorFallback.tsx

```diff
@@ -10,13 +10,19 @@ export const CustomErrorFallback: FC<CustomErrorFallbackProps> = ({ error }) =>
     const title = 'Something went wrong:';
     const reloadButton = 'Reload page';
     const returnHomeButton = 'Return to Home page';
+    const handleReload = () => {};
+    const handleReturn = () => {};
```

Comment:
this should be:
`window.location.reload()`
and
`window.location.replace('/')`